beacon-chain/rpc/beacon/validators.go

```diff
@@ -199,16 +199,54 @@ func (bs *Server) ListValidators(
 	}
 
 	validatorList := make([]*ethpb.Validators_ValidatorContainer, 0)
-	for i := 0; i < headState.NumValidators(); i++ {
-		val, err := headState.ValidatorAtIndex(uint64(i))
+
+	for _, indice := range req.Indices {
```

Comment:
```suggestion
	for _, index := range req.Indices {
```

I think index is more popular among our codebase so let's stick with that convention